src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/DraftAdjudicationController.kt

```diff
@@ -92,13 +91,13 @@ class DraftAdjudicationController {
   @PostMapping(value = ["/{id}/incident-statement"])
   @PreAuthorize("hasAuthority('SCOPE_write')")
   @ResponseStatus(HttpStatus.CREATED)
-  fun addIncidentStatement(
+  fun editIncidentDetails(
```

Comment:
Name should be for statement not details and I think it is adding not editing

Reply:
You're 100% correct. It should be called **addIncidentStatement**